packages/docs/src/examples/v-text-field/prop-counter.vue

```diff
@@ -35,10 +35,11 @@
         >
           <v-text-field
             v-model="title"
-            :rules="rules"
-            counter="25"
-            filled
-            label="Filled"
+            :rules="wordsRules"
+            counter="5"
+            hint="This field counts words instead of characters"
+            label="Custom counter from prop"
+            :counter-value="v => v.trim().split(' ').length"
           ></v-text-field>
```

Comment:
`/>` ?

Reply:
```suggestion
          />
```

Reply:
Self-closing tags don't work in DOM templates so we don't use them in examples.